---
layout: post
title: Research
update_date: 2016-05-01
date: 2002-01-01
summary: The current focus of my research is to (i) understand the relationship between sequencing depth and statistical power in sequencing studies and to (ii) improve multiple hypothesis testing for high-dimensional data.
permalink: /Research/
logo: area-chart
---


<!--* Experimental design in sequencing studies
  - [Predicting adequate read depth in sequencing studies to maximize statistical power](#predicting-adequate-read-depth-in-sequencing-studies-to-maximize-statistical-power)
* Multiple testing in high-dimensional data
  - [Optimal discovery procedure for generalized study designs](#optimal-discovery-procedure-for-generalized-study-designs)
  - Optimally combining p-values for multiple testing
--->

### Optimal discovery procedure for generalized study designs
**Collaborators**: [John D. Storey](http://genomine.org)

*Under construction*


### Predicting adequate read depth in sequencing studies to maximize statistical power

A common problem when designing a sequencing experiment is determining how many reads to sequence. Usually researchers will try to find a similar experiment (which can be difficult) or run a pilot study to guide their choice in read depth. In either case, subsampling methodologies are applied to assess whether more reads should be sequenced to maximize statistical power in the experiment. However, these methodologies do not provide guidelines for how many additional reads should be sequenced. We have developed a parametric model, superSeq, to predict the increase in power from an increasing sequencing depth. Therefore, researchers can use our model to determine how many reads, if any, need to be sequenced to maximize statistical power in a completed sequencing experiment.

#### Presentations 

- Icahn Think & Drink (2015): [pdf](http://ajbass.github.io/superSeq_icahn.pdf)

#### Posters

- John W. Tukey 100th Birthday Celebration (2015): [pdf](http://ajbass.github.io//bioc_poster.pdf)
- Bioconductor (2015): [pdf](http://ajbass.github.io//bioc_poster.pdf)

#### Collaborators
- [David G. Robinson](http://varianceexplained.org/)
- [John D. Storey](http://genomine.org)


[1]: http://ajbass.github.io/R-packages
[2]: http://ajbass.github.io/andrewbass_cv.pdf
[3]: http://ajbass.github.io/Projects
